automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceSpec.scala

```diff
@@ -335,43 +335,47 @@ class WorkspaceSpec extends FreeSpec with WebBrowserSpec with WorkspaceFixtures
 
     "who has writer access" - {
       "and does not have canCompute permission" - {
-        "should see launch analysis button disabled" ignore withWebDriver { implicit driver =>
-          val Seq(user1, user2) = UserPool.chooseStudents(2)
-          implicit val authToken: AuthToken = Config.Users.owner.makeAuthToken()
-          withWorkspace(billingProject, "WorkspaceSpect_writerAccess") { workspaceName =>
-            withSignIn(user1) { listPage =>
-              api.methodConfigurations.createMethodConfigInWorkspace(billingProject, workspaceName, SimpleMethod, SimpleMethodConfig.configNamespace, s"$methodConfigName", 1,
+        "should see launch analysis button disabled" in withWebDriver { implicit driver =>
+          val user = UserPool.chooseStudent
+          val owner = UserPool.chooseProjectOwner
+          implicit val authToken: AuthToken = authTokenOwner
+          val testName = "WorkspaceSpec_writerAccess_withCompute"
+          withMethod(testName, MethodData.SimpleMethod) { methodName =>
+            withWorkspace(billingProject, testName) { workspaceName =>
+              withSignIn(owner) { listPage =>
+              api.methodConfigurations.createMethodConfigInWorkspace(billingProject, workspaceName, MethodData.SimpleMethod.copy(methodName = methodName),
+                SimpleMethodConfig.configNamespace, s"$methodConfigName", 1,
                 SimpleMethodConfig.inputs, SimpleMethodConfig.outputs, "participant")
-              api.methodConfigurations.setMethodConfigPermission(SimpleMethodConfig.configNamespace, s"$methodConfigName", 1, user1.email, "OWNER")
               val detailPage = listPage.enterWorkspace(billingProject, workspaceName)
-              detailPage.share(user2.email, "WRITER", false, false)
-            }
-            withSignIn(user2) { listPage2 =>
-              val detailPage2 = listPage2.enterWorkspace(billingProject, workspaceName)
-              val methodConfigTab = detailPage2.goToMethodConfigTab()
-              val methodConfigDetailsPage = methodConfigTab.openMethodConfig(SimpleMethodConfig.configNamespace, s"$methodConfigName")
-              val errorModal = methodConfigDetailsPage.clickLaunchAnalysisButtonError()
-              errorModal.getErrorText shouldBe "You do not have access to run analysis.\nCancel"
+              detailPage.share(user.email, "WRITER", false, false, Some(true))
```

Comment:
Ideally, this test could use an API for this and eliminate the need for `owner` to use the UI at all. (This could be done here or in a followup ticket, not sure which would be more appropriate.) However, it would take a bit more work to handle the case where we need to share the method with `user`.